cypress/e2e/immutableDatabaseTests/newsNotifications.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ *    Copyright 2020 OICR
```

Comment:
Date, there should be versions of this with 2023 and UCSC, but only if you're going in here to fix other stuff